shopify_function_macro/src/lib.rs

```diff
@@ -173,15 +348,15 @@ fn generate_struct(name: &str, query_path: &str, schema_path: &str) -> TokenStre
     let name_ident = Ident::new(name, Span::mixed_site());
 
     quote! {
-        #[derive(graphql_client::GraphQLQuery, Clone, Debug, serde::Deserialize, PartialEq)]
+        #[derive(graphql_client::GraphQLQuery, Clone, Debug, serde::Deserialize, serde::Serialize, PartialEq)]
         #[graphql(
             query_path = #query_path,
             schema_path = #schema_path,
-            response_derives = "Clone,Debug,PartialEq,Eq,Deserialize",
-            variables_derives = "Clone,Debug,PartialEq,Eq,Deserialize",
+            response_derives = "Clone,Debug,PartialEq,Eq,Deserialize,Serialize",
+            variables_derives = "Clone,Debug,PartialEq,Eq,Deserialize,Serialize",
```

Comment:
🤔 I don't know if we want to support this, but there's a (private) [example](https://github.com/Shopify/functions-externals-project/blob/f6fa30ad0c72a532da68cd9d5a23fabf19d2b89a/tutorial/app/extensions/max-quantity-rust/src/fetch.rs#L43) that's depending on [`response_derives` having Serialize](https://github.com/Shopify/functions-externals-project/blob/f6fa30ad0c72a532da68cd9d5a23fabf19d2b89a/tutorial/app/extensions/max-quantity-rust/src/fetch.rs#L13)